doc/api/https.md

```diff
@@ -61,6 +61,12 @@ changes:
 An [`Agent`][] object for HTTPS similar to [`http.Agent`][]. See
 [`https.request()`][] for more information.
 
+Like `http.Agent`, the `createConnection(options[, callback])` method can be overridden
+to customize how TLS connections are established.
+
+> See [`http.Agent#createConnection()`][] for details on overriding this method,
```

Comment:
Is this accurate? When we were digging into things we noticed that the signature of `createConnection` was [completely different for https](https://github.com/nodejs/node/blob/main/lib/https.js#L141C10-L141C26).

Reply:
Ah, I see... it's just never called that way within `Agent.prototype.createSocket`. I do see that your docs are accurate, never mind. The code just makes it confusing because the source function is used in multiple contexts.